src/cmd/initialize.go

```diff
@@ -130,6 +138,7 @@ func downloadInitPackage(ctx context.Context, cacheDirectory string) (string, er
 	message.Note(lang.CmdInitPullNote)
 
 	// Prompt the user if --confirm not specified
+	// REVIEW(mkcp): It looks like this condition can't be met - maybe --confirm was removed at some point?
```

Comment:
<img width="719" alt="Screenshot 2024-10-01 at 10 07 02 AM" src="https://github.com/user-attachments/assets/ff42561b-300a-4e81-a492-f19be6ea51f1">
OK, I had a silly mistake here misreading the warning I was getting from goland. It was saying that the if block is unnecessary because it always runs at that point.